### PR TITLE
Constant cleanup

### PR DESCRIPTION
**Pull request by jkisor**
- Replace literals with existing constants
- Remove unused constant imports 
- Remove unused constants (* Except `SKILL_DEPENDS_ON_ATTRIBUTES` as it looks substantial and possibly useful in the future?) 
- Remove a min/max on a text input (times enlightened) -- AFAIK min/max doesn't work for text fields

I was originally interested in cleaning up some duplication in the skill investment changes and noticed that there were a mix of literals/constants for the max values of trained vs specialized. I decided to have it all depend on the existing constants and then continued with assessing some of the other constants as well.

I'll add some inline comments to point out some decisions.

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -11,10 +11,8 @@ import {
 
 export const MIN_LEVEL: number = 1;
 export const MAX_LEVEL: number = 275;
-export const MIN_TIMES_ENLIGHTENDED: number = 0;
```
Comment (jkisor) [inline src/constants.ts:14]: This was not used anywhere, other than an import that wasn't used. We _could_ use it in the range input, but things like 0 or 1 seem like more obvious literals than maximums. Introducing minimum constants for other ranges seems tedious and low-value add (given the song and dance needed to get it available to the template)

File: src/constants.ts [hunk 2]
```diff
@@ -11,10 +11,8 @@ import {
...
 export const MAX_TIMES_ENLIGHTENDED: number = 5;
 export const MAX_SPECIALIZED_SKILL_CREDITS_SPENT: number = 70;
-export const MAX_CREATION_ATTRIBUTE_POINTS: number = 100;
```
Comment (jkisor) [inline src/constants.ts:17]: no references found.

File: src/components/Headers.vue [hunk 3]
```diff
@@ -128,14 +128,12 @@
...
             <div class="w70">
-              <input type="range" min="0" max="5" v-model="timesEnlightened" />
+              <input type="range" min="0" :max="maxTimesEnlightened" v-model="timesEnlightened" />
             </div>
             <div class="w30 right">
               <input
                 class="number"
                 type="text"
-                min="0"
-                max="5"
```
Comment (jkisor) [inline src/components/Headers.vue:138]: min/max don't seem to work with text input

File: src/constants.ts (hunk 2 above)
Comment (jkisor) [reply src/constants.ts:17]: nevermind, I need to rebase -- this was made useful recently.

**Comment by amoeba** [comment]
Hey @jkisor thanks for going through the code and doing this. I definitely did not keep to a good practice on using constants.

This looks good to me as-is.

**Pull request merged**